README.md

```diff
@@ -3,9 +3,19 @@ WebLogic Monitoring Exporter
 
 [![Build Status](https://travis-ci.org/oracle/weblogic-monitoring-exporter.svg?branch=master)](https://travis-ci.org/oracle/weblogic-monitoring-exporter)
 
-The WebLogic Monitoring Exporter uses the [WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111) to scrape runtime information and then exports [Prometheus](http://prometheus.io)-compatible metrics.
-It is available both as a [web application](#web-application) to be deployed to the WebLogic Server (WLS) instance, version 12.2.1 or later, from which you want to get metrics, 
-and also as a [sidecar](#sidecar), which runs as its own process, and which is configured to contact a WLS instance.
+The WebLogic Monitoring Exporter is a [Prometheus](http://prometheus.io)-compatible exporter of metrics from 
+WebLogic Server (WLS) instances, which it obtains by using the 
+[WLS RESTful Management API](https://docs.oracle.com/middleware/12213/wls/WLRUR/overview.htm#WLRUR111), available in version 12.2.1 or later. 
+Metrics are selected using a [YAML configuration file](#Configuration).
+
+The exporter is available in two forms:
+ - A [web application](#web-application) that you deploy to the Managed Server from which metrics are to be extracted. 
+ You may include a configuration file directly in the WAR file, and you may temporarily modify the configuration in a 
+ running system by using a web form. If a [coordination configurator](config_coordinator/README.md) is running and configured, 
+ that temporary configuration will be sent to all servers configured to use it.
```

Comment:
Would the following be more precise?

You may embed a configuration file directly in the web application's WAR file or configure the web-application to obtain its configuration from a central [coordination configurator](config_coordinator/README.md).  In addition, you can temporarily modify the configuration dynamically by using a web form in your browser, and if a coordination configurator is setup, then any web-form submissions will be propagated to all servers that use the coordinator.

Reply:
No, that's not actually correct at all.

Reply:
Then can you make the original wording more helpful? It's not clear to me (at least) how configuration initialization and configuration changes are submitted to the coordinator.  This is nice information to have in an introduction.